### PR TITLE
Add Hire us section to docs

### PR DESCRIPTION
**Pull request by patrycjakalinska**
image

We added `Hire us` section to help clients recognise and contact us directly from documentation's landing page.

Additionally we expanded `README.md` by adding information about Discord and SWM.

### CODE REVIEW

File: docs/src/components/HireUsSection/index.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import HomepageButton from '../HomepageButton';
+import styles from './styles.module.css';
+
+const HireUsSection = () => {
+  return (
+    <div className={styles.hireUsSectionWrapper}>
+      <div className={styles.hireUsTitleContainer}>
+        <h2>We are Software Mansion</h2>
```
Comment (kacperkapusciak) [inline docs/src/components/HireUsSection/index.tsx:9]:
```suggestion
        <h2>We are Software Mansion.</h2>
```

File: docs/src/components/HireUsSection/styles.module.css [hunk 2]
```diff
@@ -0,0 +1,71 @@
+.hireUsSectionWrapper {
+  display: flex;
+  position: relative;
+  flex-direction: column;
+  align-items: center;
+  gap: 2.5rem;
+  width: 60%;
+  margin: 9.5rem auto;
```
Comment (kacperkapusciak) [inline docs/src/components/HireUsSection/styles.module.css:8]: lets add more margin on the top and bottom

Comment (kacperkapusciak) [reply docs/src/components/HireUsSection/styles.module.css:8]: maybe 12rem top and 16rem bottom, or something like that

File: README.md [hunk 3]
```diff
@@ -49,3 +49,11 @@ This project has been built and is maintained thanks to the support from [Shopif
...
 [![expo](https://avatars2.githubusercontent.com/u/12504344?v=3&s=100 'Expo.io')](https://expo.io)
 [![swm](https://logo.swmansion.com/logo?color=white&variant=desktop&width=150&tag=react-native-reanimated-github 'Software Mansion')](https://swmansion.com)
+
+## Community Discord
+
+[Join the Software Mansion Community Discord](https://discord.swmansion.com) to chat about Reanimated or other Software Mansion libraries.
+
+## Reanimated is created by Software Mansion
+
+[Software Mansion](https://swmansion.com) is a software agency with over 11 years of experience in building web and mobile apps. We are Core React Native Contributors and experts in dealing with all kinds of React Native issues. We can help you build your next dream product – [Hire us](https://swmansion.com/contact#contact-form).
```
Comment (kacperkapusciak) [inline README.md:59]: lets do "since 2012" rather than over 11 years as it will be easier to maintain in README

**Review by kacperkapusciak — APPROVED** [review]

File: docs/src/components/HireUsSection/styles.module.css [hunk 4]
```diff
@@ -0,0 +1,71 @@
+.hireUsSectionWrapper {
+  display: flex;
+  position: relative;
+  flex-direction: column;
+  align-items: center;
+  gap: 2.5rem;
+  width: 60%;
+  margin: 16rem auto 12rem auto;
+  padding-bottom: 2rem;
```
Comment (kacperkapusciak) [inline docs/src/components/HireUsSection/styles.module.css:9]:
```suggestion
  padding-bottom: 2rem;
  text-wrap: balance;
```
🙏 

image

**Review by piaskowyk — APPROVED** [review]

**Pull request merged**